tests/cases/init_tdedbserver/playbook.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: all
+  name: Postgres deployment playbook
+  become: true
+  gather_facts: true
+  any_errors_fatal: true
+
+  collections:
+    - edb_devops.edb_postgres
+
+  pre_tasks:
+    - name: Initialize the user defined variables
+      set_fact:
+        disable_logging: false
+        edb_enable_tde: true
+        edb_master_key_location: "/etc/edb/keyloc"
+        edb_master_key_location_file: "{{ edb_master_key_location }}/master_key.plain"
+        edb_key_unwrap_cmd: 'openssl enc -d -aes-128-cbc -pbkdf2 -in "%p" -pass file:/etc/edb/keyloc/master_secure_key.key'
+        edb_key_wrap_cmd: 'openssl enc -e -aes-128-cbc -pbkdf2 -in  /etc/edb/keyloc/master_key.plain -out "%p" -pass file:/etc/edb/keyloc/master_secure_key.key'
+        edb_master_key: "ItsImportantToCheckThisItsImportantToCheckThisItsImportantToCheckThis"
+        edb_secure_master_key: "12345678"
+        edb_secure_master_key_file: "{{ edb_master_key_location }}/master_secure_key.key"
+        pg_type: "EPAS"
+        pg_version: 15
```

Comment:
to be passed in by `test-runner.py`

Reply:
Not following you.

Reply:
The playbook for the test case shouldn't define the `pg_version` and `pg_type` variables, in case of future release, for example, EPAS 16, this will make the test case locked in. The `pg_version` and `pg_type` variables are also passed in through the `test-runner.py` script as environment vars, so this is redundant anyways. The configuration for which `pg_version` and `pg_types` can run the test is also done in `/tests/config.yml`.

Reply:
Got it

Reply:
Fixed